package.json

```diff
@@ -5,55 +5,57 @@
   "repository": "https://github.com/getlift/lift",
   "description": "Lift",
   "dependencies": {
-    "aws-cdk-lib": "^2.215.0",
-    "chalk": "^4.1.1",
-    "change-case": "^4.1.2",
+    "aws-cdk-lib": "^2.233.0",
+    "chalk": "^4.1.2",
+    "change-case": "^5.4.4",
     "cidr-split": "^0.1.2",
-    "constructs": "10.2.20",
-    "inquirer": "^7.3.3",
+    "constructs": "^10.4.4",
+    "inquirer": "^8.2.7",
```

Comment:
When upgrading a major version did you read the changelog (of change-case and inquirer for example) and update our code to address the breaking changes?